vernac/himsg.ml

```diff
@@ -1346,11 +1346,11 @@ let explain_prim_token_notation_error kind env sigma = function
   | Notation.UnexpectedTerm c ->
     (strbrk "Unexpected term " ++
      pr_constr_env env sigma c ++
-     strbrk (" while parsing a "^kind^" notation."))
+     strf " while parsing a %s notation." kind)
```

Comment:
You changed `strbrk` to `str` so the output test needs updating.
Or maybe there should be a `strbrkf`?

Reply:
Good point, indeed I'm not very happy about the semantics of `strbrk` [in the sense that this should be transparent from the point of view of API]

I'll try to revise this.